concourse_web_config.tf

```
@@ -63,6 +68,7 @@ locals {
     var.proxy.https_proxy != null ? { HTTPS_PROXY = var.proxy.https_proxy, https_proxy = var.proxy.https_proxy } : {},
     var.proxy.no_proxy != null ? { NO_PROXY = var.proxy.no_proxy, no_proxy = var.proxy.no_proxy } : {},
     //  var.web.environment_override
+    local.web_service_env_vars_saml
```

rather than a separate local, can you put the SAML stuff here to be consistent with how the proxy ones are done?

---

I can. However I don't think it's a better approach because:
- instead of evaluating `var.concourse_saml_conf.enable_saml` once, it'd be done for every var. Not neat.
- code will be harder to read as lines will be long, e.g.
```
var.concourse_saml_conf.enable_saml ? CONCOURSE_SAML_DISPLAY_NAME  = var.concourse_saml_conf.display_name : {}
```

---

no, not for each one, just copy everything to the right of the `web_service_env_vars_saml = ` and put it here
e.g.:
```suggestion
    var.concourse_saml_conf.enable_saml ? {
    # SAML Auth
    CONCOURSE_SAML_DISPLAY_NAME  = var.concourse_saml_conf.display_name
    CONCOURSE_SAML_SSO_URL       = var.concourse_saml_conf.url
    CONCOURSE_SAML_CA_CERT       = local.okta_cert_path
    CONCOURSE_SAML_SSO_ISSUER    = var.concourse_saml_conf.issuer
    CONCOURSE_SAML_USERNAME_ATTR = var.concourse_saml_conf.concourse_saml_username_attr
    CONCOURSE_SAML_EMAIL_ATTR    = var.concourse_saml_conf.concourse_saml_email_attr
    CONCOURSE_SAML_GROUPS_ATTR   = var.concourse_saml_conf.concourse_saml_groups_attr
  } : {},
```

---

the reason I split the proxy one is because you can supply each of the three independently of the others, e.g. you might want a `no_proxy` setting, but route everything else via internet